test/integration-test/CodeGen/hd_smagorinsky_benchmark.cpp

```diff
@@ -77,6 +77,7 @@ TEST(hd_smagorinsky, test) {
   cxxnaive::hd_smagorinsky_stencil hd_smagorinsky_naive(dom, u_out_naive, v_out_naive, u_in, v_in, hdmaskvel, crlavo, crlavu,
                                                         crlato, crlatu, acrlat0, eddlon, eddlat, tau_smag,
                                                         weight_smag);
+  verif.sync_storages(u_in, v_in, hdmaskvel, crlavo, crlavu, crlato, crlatu, acrlat0, eddlon, eddlat, tau_smag, weight_smag);
```

Comment:
we should stick to one way of doing it:

```
stencil s_gt(...)
stencil s_cxx(...)

s_gt.run()
verif.sync(input_fields)
s_cxx.run()
```
or 
```
stencil s_gt(...)
s_gt.run()

verif.sync(input_fields)
stencil s_cxx(...)
s_cxx.run()
```

I prefer the first one

Reply:
done